src/test/crimson/seastore/test_transaction_manager.cc

```diff
@@ -487,6 +487,34 @@ struct transaction_manager_test_t :
     return ext;
   }
 
+  TestBlockRef try_alloc_extent_by_pin(
```

Comment:
See the other comment. It is meaningless to put `tm->read_extent()` and `test_mappings.alloced()` together.

Move `test_mappings.alloced()` out and rename the method as `read_pin()`. Probably it is better to call `tm->read_pin()` instead.